src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.ts

```diff
@@ -154,6 +160,7 @@ export class ExerciseAssessmentDashboardComponent implements OnInit {
         private guidedTourService: GuidedTourService,
         private artemisDatePipe: ArtemisDatePipe,
         private sortService: SortService,
+        private javaBridge: OrionConnectorService,
```

Comment:
Why is this called javaBridge but not orion connector for example?

Reply:
I must admit I was following the example of other places in the code without thinking too much about it; I will refactor that.